api/test/service/store.test.js

```diff
@@ -15,21 +16,28 @@ import { createS3, createBucket, createDynamodDb, createAccessServer } from '../
  * @typedef {import('../../service/types').ListResponse<StoreListResult>} ListResponse
  */
 
-test.beforeEach(async t => {
-  const region = 'us-west-2'
-  const tableName = 'store'
+const REGION = 'us-west-2'
```

Comment:
I think we can just set a default region for the tests and not worry about. We don't need to pass it around, minio don't care about it. I would remove all references of REGION from the tests.
```suggestion
```